sources/packages/backend/apps/api/src/services/student/student.service.ts

```diff
@@ -228,6 +230,14 @@ export class StudentService extends RecordDataModelService<Student> {
         await entityManager
           .getRepository(SFASRestriction)
           .update({ individualId: sfasIndividual.id }, { processed: false });
+
+        // If there is a match in SFAS, record that information is available from legacy system with a note attached to the student.
+        this.addStudentNote(
```

Comment:
Please use the method `noteSharedService.createStudentNote`  that would allow the execution in the same transaction `entityManager`.

Reply:
updated